### ISSUE TITLE
pydocstyle can't parse a function when *args is specified before argument with default value

### ISSUE THREAD

**Issue by oerp-odoo** [PyCQA/pydocstyle#350]
For some reason pydcostyle fails, when trying to parse file that has this function:

```python
def _func(a, b, *args, c=5, **kwargs):
    pass
```

And problem is with `*args` specified before `c=5`. If I remove `*args` or `c=5` (or swap it in places), then it parses the file.

Using pydocstyle 3.0.0

**Comment by sigmavirus24** [comment]
What version of python are you running pydocstyle on? I wonder if this is due to the stdlib and not being able to tokenize that as it's Python3 syntax.

**Comment by oerp-odoo** [comment]
@sigmavirus24 `Python 3.5.2`

**Comment by sigmavirus24** [comment]
So to be clear, doing `python3.5 -m pydocstyle file.py` fails to parse the file? Can you give the full stack trace?

**Comment by oerp-odoo** [comment]
@sigmavirus24 false alarm. I thought I was running with python3, but actually pydocstyle was running with python2 (I was using command `pydocstyle` without specifying which python version to run).

`pydocstyle file.py` fails.
`python2 -m pydocstyle file.py` fails
`python3.5 -m pydocstyle file.py` passes.

**Comment by sigmavirus24** [comment]
No worries. I thought that might be the case. Thanks for working through that with me. :tada:
